docs/source/glossary.rst

```diff
@@ -0,0 +1,134 @@
+        detail. The same model can often be used to describe multiple similar
+        :term:`instruments<instrument>` or multiple :term:`versions<version>` of an
+        :term:`instrument`, but in the context of this package, a model can only be accessed through
+        a particular :term:`version` of an :term:`instrument`.
+
+
+Neutron terminology
+^^^^^^^^^^^^^^^^^^^
+
+.. glossary::
+
+    source
+        A device that produces neutrons used for neutron experiments. In practice, this is usually
+        either a nuclear reactor or spallation source (a particle accelerator).
+
+    target
+        A material used in spallation :term:`sources<source>`, which is hit by the particle produced
+        from the accelerator. The collision emits neutrons which are used for the experiments.
+
+    moderator
+        A material used to slow down the neutrons produced by the collision with the :term:`target`.
+
+    beam
+        A collection of neutrons. In nuclear reactor :term:`sources<source>`, this is usually a
+        continuous beam (somewhat similar to a laser beam), while in spallation
+        :term:`sources<source>` this is usually a pulse (a short burst).
+
+    chopper
+        A rotating mechanical device designed to block the neutron :term:`beam` for some fraction of
+        each revolution.
+
+    Fermi chopper
+        A :term:`chopper` designed to select a particular narrow slice of energies from the wide
+        range of energies coming from the moderator. This kind of :term:`chopper` often has multiple
+        openings, allowing the user to make a choice of which energies to use or how good a
+        :term:`resolution` to obtain.
+
+    disk chopper
+        A type of :term:`chopper` shaped like a disk. Can be used for various purposes, such as
+        improving :term:`resolution` or removing contamination, etc.
+
+    sample
+        The material being studied by the neutron experiment. Unless specified otherwise, in this
+        library the term "sample" combines two things: the :term:`sample environment` and the sample
+        itself. The latter is the stuff that the user brings to be studied, in :term:`INS` usually
+        a single crystal or a powder.
+
+    sample environment
+        A device used to enclose the :term:`sample`, usually an aluminium "can". May consist of
+        extra parts, such as a cooling system. Further parts may also be present, such as a vacuum
+        pump or extra experimental devices like a Raman spectrometer, but these are generally not
+        exposed to the neutron :term:`beam` and therefore shouldn't affect the :term:`resolution`.
+
+    detector
+        A device used used to detect the presence of a neutron. These are placed in an array such
+        that the the neutrons that hit the :term:`sample`, hit the detectors, registering where and
+        when the neutron hit the detector.
```

Comment:
In indirect-geometry instruments, scattered neutrons will only reach a detector via an energy-selective "analyzer".

Reply:
Do you want to add that as an extra sentence at the end?

Reply:
I think the existing sentence needs reworking. Bear in mind that not _all_ the neutrons that "hit" the sample are scattered, and not all the scattered neutrons will go in an appropriate direction to reach an analyser and/or detector.

Reply:
```
    detector
        A device used used to detect the presence of a neutron. These are placed at various
        positions around the sample to determine the energies and momenta of the neutrons
        scattered from the sample under investigation.
```
?

Reply:
I'm struggling to write something much better than this, so let's use it for now!